packages/cubejs-client-core/src/tests/drill-down.test.js

```diff
@@ -123,7 +123,7 @@ describe('drill down query', () => {
       timeDimensions: [
         {
           dimension: 'Orders.ts',
-          dateRange: ['2020-08-01T00:00:00.000', '2020-08-01T23:59:59.999'],
+          dateRange: ['2020-08-01T00:00:00.000', '2020-08-01T23:59:59.000'],
```

Comment:
Is there any reason you truncate time here? Feels like it may lead to unpredicted behavior.

Reply:
Sorry for the careless mistake, I thought this was a semantic change in how dayjs parses but realized I did in fact use the wrong stringify method.